manifests/server/install.pp

```diff
@@ -1,5 +1,7 @@
 # == Class dns::server
 #
+include dns::server::default
```

Comment:
@solarkennedy I've included it:
`include dns::server::default`

Now everything should be fine. If so, you need a rebase?